config/helm/certificate.yaml

```diff
@@ -0,0 +1,45 @@
+---
```

Comment:
Changing DNS names (to set our namespace) and issuer reference (to use our existing issuer).